lib/daos/tests/userSubjectsDAO.test.js

```diff
@@ -0,0 +1,21 @@
+import { mockData } from 'utils/mockData';
+describe('userSubject daos', () => {
+    const { MOCK_USER_SUBJECTS: mockUserSubjects } = mockData;
+    describe('findOneUserSubject', () => {
+        it('should find a userSubject by ID', async () => {
+            const { findOneUserSubject } = require('daos/userSubjectsDao');
+            const testSubject = await findOneUserSubject(1);
+            expect(testSubject.id).toEqual(1);
```

Comment:
mockSubject.id. This is try for all. If we are reusing this in a lot of places probably makes sense to rollup in a function